src/ErrorData.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+export class ErrorData {
+    public readonly message: string;
+    public readonly errorType: string;
+    // tslint:disable-next-line:no-any
+    constructor(error: any) {
+        if (error instanceof Error) {
+            try {
+                this.errorType = JSON.parse(error.message).Code;

```

Comment:
Just curious - what's an example value for 'Code'? What does it look like?

Reply:
From what I can tell, it's a one line word describing the error.  For instance, if you try to deploy two slots to production, the code will come back as "Conflict".